versioned_docs/version-1.1.0/alert-design/custom-alert-templates.md

```diff
@@ -0,0 +1,152 @@
+# Custom Alert Templates
+
+<!-- https://docs.google.com/document/d/1-8K_nbJGaywL-jYFg7_YZiE9x_WPaHvuFfYgM7QmqMg/edit -->
+
+Notifi uses templates to render the same notification in different forms for different channels such as email, Telegram, and SMS. We use [Mustache templates](https://mustache.github.io/mustache.5.html), and you can try out a demo here: https://mustache.github.io/#demo
```

Comment:
the problem with this is that we're expecting {{{ }}} in a few templates... idk which, but I know a few of mine need triple while others needs doubles

Reply:
Yeah, I can address this in the new docs